presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4117,20 +4118,37 @@ public void testPredicatePushDownToTableScan()
     }
 
     @Test
-    public void testParquetTimestampPredicatePushdown()
+    public void testParquetTimestampInt96PredicatePushdown()
     {
-        assertUpdate("CREATE TABLE test_parquet_timestamp_predicate_pushdown (t TIMESTAMP) WITH (format = 'PARQUET')");
-        assertUpdate("INSERT INTO test_parquet_timestamp_predicate_pushdown VALUES (TIMESTAMP '2012-10-31 01:00')", 1);
+        testParquetTimestampPredicatePushdown(getSession());
+    }
+
+    @Test
+    public void testParquetTimestampInt64PredicatePushdown()
```

Comment:
Will need to be updated when we do https://github.com/prestosql/presto/issues/4608 since the test assumes the new writer uses int64 timestamps by default.